Reflections/Week_4_reflection.md

```diff
@@ -0,0 +1,48 @@
+# Reflection 4
+Jose Manuel Cortes
+
+*04/30/2021*
+
+---
+
+1. What have you accomplished this week? (Please list each accomplishment, and explain)
+
+    * Hardware
+
+      I managed to fix the reconnection issue in which the hardware would be dropped from the network upon a network drop/reboot. I had implemented a function to check if network status was dropped or disconnected and to begin attempting a reconnection every 10 seconds till connection was established. All other functions would cease till connection was established. I also updated the comments on the C++ code to me transparent and descriptive of said code. I also moved around more user definable variables into a file called "settings.h" so if people can define network credentials, location/sensor number, etc... Testing has been successful so far and I believe hardware portion in my opinion to this point is complete for a initial release till more expansive setups could be created for further testing.
```

Comment:
Congrats on completing the hardware section of your project and on successfully solving the problem of the network disconnecting! The next step would be carrying out a unit test on your hardware and integration test when you combine it with other features. Great job!